RELEASE_NOTES.md

```diff
@@ -1,8 +1,9 @@
 # Release notes
 
-## New in git-machete 3.9.2
+## New in git-machete 3.10.0
 
-- added: `git machete github *` subcommands now work with custom GitHub URL
+- added: 3 git config keys `machete.github.{remote,organization,repository}` that enables `git machete github *` subcommands to work
+  with custom GitHub URLs
```

Comment:
Crap, I'm not sure if magic script that generates a changelog for Debian from RELEASE_NOTES.md will work properly with a broken line (the Debian changelog format is brittle as hell)... pls merge to the line above